webrender/res/prim_shared.glsl

```diff
@@ -250,9 +258,14 @@ float do_clip() {
     if (!all(inside)) {
         return 0.0;
     }
+
     // finally, the slow path - fetch the mask value from an image
-    ivec3 tc = ivec3(mask_uv, vClipMaskUv.z);
-    return texelFetch(sCacheA8, tc, 0).r;
+
+    // TODO(gw): texelFetch here fails on some nVidia hardware in
```

Comment:
this is really really sad :crying_cat_face:

Reply:
Yes :cry: